packages/components/src/patterns/supernumber/supernumber.stories.ts

```diff
@@ -14,14 +14,76 @@ export default {
 };
 
 /**
- * Used to highlight and convey important key figures. The component consists of overline, number, subline and description. Supernumber is used as a subcomponent inside sd-container.
+ * Used to highlight and convey important key figures. The pattern consists of overline, number, subline and description. Supernumber is used as a subcomponent inside sd-container.
+ *
+ * Supernumbers are preferably entered in the form of numbers, but can also be entered in the form of text in order to fulfill certain use cases.
  */
 
 export const Default = {
   render: () => html`
-    <div class="sd-container sd-container--variant-white flex flex-col items-center">
+    <div class="sd-container sd-container--variant-white flex flex-col items-center text-center">
+      <div class="text-[72px] text-primary leading-[86.4px]">XXX</div>
+    </div>
+  `
+};
+
+/**
+ * The supernumber can be displayed with description. The description is displayed below the number, or (if exists) suffix.
+ */
+
+export const Description = {
+  name: 'Description (optional)',
+  render: () => html`
+    <div class="sd-container sd-container--variant-white flex flex-col items-center text-center">
+      <div class="text-[72px] text-primary leading-[86.4px]">XXX</div>
+      <p class="sd-pararaph sd-paragraph--size-sm text-base text-center pt-2">
+        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmd tempor invit ut labore et dolore
+        magna aliquyam erat
+      </p>
+    </div>
+  `
+};
+
+/**
+ * The supernumber can be displayed with a prefix. The overline is displayed above the number.
+ */
+
+export const Overline = {
+  name: 'Overline (optional)',
+  render: () => html`
+    <div class="sd-container sd-container--variant-white flex flex-col items-center text-center">
+      <p class="sd-paragraph font-bold mb-2">Nisi eu excepteur anim esse</p>
+      <div class="text-[72px] text-primary leading-[86.4px]">XXX</div>
+    </div>
+  `
+};
+
+/**
+ * The supernumber can be displayed with a Subline. The subline is displayed below the number.
```

Comment:
* The supernumber can be displayed with a subline. The subline is displayed below the number.